requirements.txt

```diff
@@ -1,2 +1,4 @@
-nwb-conversion-tools==0.11.1
+nwb-conversion-tools>0.11.1
```

Comment:
```suggestion
nwb-conversion-tools @ git+https://github.com/catalystneuro/nwb-conversion-tools.git@a0b2d5af51edd8d20e9c05bd5ded02cd68485cc2  # >= 0.11.26 once we get a new release
```

Reply:
(Since it's still on a dev branch, safest thing to do is pin to commit hash until new release)

Reply:
Ah, I see. Thanks!